rpcs3/Emu/Cell/Modules/cellAudio.cpp

```diff
@@ -1650,18 +1652,45 @@ error_code cellAudioCreateNotifyEventQueueEx(ppu_thread& ppu, vm::ptr<u32> id, v
 	return AudioCreateNotifyEventQueue(ppu, id, key, queue_type);
 }
 
-error_code AudioSetNotifyEventQueue(u64 key, u32 iFlags)
+error_code AudioSetNotifyEventQueue(ppu_thread& ppu, u64 key, u32 iFlags)
 {
 	auto& g_audio = g_fxo->get<cell_audio>();
 
+	if (!g_audio.init)
+	{
+		return CELL_AUDIO_ERROR_NOT_INIT;
+	}
+
+	// Waiting for VSH
+	lv2_sleep(10'000, &ppu);
```

Comment:
What if I'm not using vsh?

Reply:
I mean that is what this function does on PS3.